provisioning_templates/provision/kickstart_default.erb

```diff
@@ -40,7 +40,9 @@ This template accepts the following parameters:
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
+<% if @host.operatingsystem.name != 'Fedora' || os_major < 29 -%>
 install
+<% end -%>
```

Comment:
Not sure if the statement is correct, what OS/version combination must not have this statement? To me this smells better: `<% if @host.operatingsystem.name == 'Fedora' && os_major >= 28 -%>`

Reply:
Starting Fedora 29 it shouldn't be there, but until then I think it should. The alternative is `<% unless @host.operatingsystem.name == 'Fedora' && os_major >= 29 -%>` which may be a bit easier to read.